src/components/modules/Learn/index.js

```diff
@@ -0,0 +1 @@
+export { default } from ".Learn.js";
```

Comment:
Looks like there's a missing slash between `.` and `Learn.js`

Reply:
Fixed.